VSRAD.Package/DebugVisualizer/VisualizerTable.cs

```diff
@@ -302,19 +308,29 @@ private static void LockWatchRowForEditing(DataGridViewRow row, bool canBeRemove
 
         #region Styling
 
-        public void GrayOutColumns(uint groupSize)
+        public void GrayOutColumns()
         {
-            _computedStyling.GrayOutColumns(groupSize);
```

Comment:
This looks like the only usage of `ComputedColumnStyling.GrayOutColumns`, you can remove it. I don't think `ColumnStyling.GrayOutColumns` is used anywhere either, that can be removed too.

Reply:
Thanks! Resolved in c1dce41